.github/workflows/action.yml

```diff
@@ -0,0 +1,21 @@
+name: "test"
+
+on:
+  pull_request:
```

Comment:
[question] is this a holdover from a past action? i'm guessing we don't want the deploy action running on `pull_request`?

Reply:
It's a bad way to do a real deploy, but here it is making sure that the action works on a PR to this repo. This file is not included when other repos use this shared action.